tests/unit/src/test/scala/org/scalafmt/tests/TermMatchSuite.scala

```diff
@@ -10,5 +10,4 @@ object TermMatchSuite extends BaseScalaPrinterTest {
   check("a match { case 1 => }")
   check("(a: @switch) match { case _ => b }")
   check("(if (a) b else c) match { case 1 => }")
-  check("a match {case _ => { f op { b } } }")
```

Comment:
Why is this test removed? If the output has changed we can update the test. the test is there only for ast changes

Reply:
I think the ci is disabled, this should not have passed.